From e05b2f79c22f11ec218aa373806867f89cfa0b76 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Tue, 4 Dec 2018 14:19:06 -0800
Subject: [PATCH] MANTA-4033 get lagging Manta components on the latest v0.10.x
 Reviewed by: Joshua M. Clulow <jmc@joyent.com> Approved by: Joshua M. Clulow
 <jmc@joyent.com>

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 5016887..1e87737 100644
--- a/Makefile
+++ b/Makefile
@@ -35,7 +35,7 @@ JSSTYLE_FILES	 = $(JS_FILES)
 # Variables
 #
 NAME			= marlin-dashboard
-NODE_PREBUILT_VERSION	= v0.10.25
+NODE_PREBUILT_VERSION	= v0.10.48
 NODE_PREBUILT_TAG	= zone
 NODE_PREBUILT_IMAGE	= fd2cc906-8938-11e3-beab-4359c665ac99
 
-- 
2.21.0

